cmd/provisioner-localpv/main.go

```diff
@@ -32,9 +43,8 @@ func main() {
 
 // run starts the dynamic provisioner for Local PVs
 func run() error {
-	// Init logging
-	mlogger.InitLogs()
-	defer mlogger.FlushLogs()
+	// Flush logs
+	defer logger.FinishLogging()
```

Comment:
Its better to have the initalization and defer close to each other.

Reply:
Init and deferred finish functions are now close to each other in the run() function.